src/main/webapp/web.xml

```diff
@@ -7,6 +7,46 @@
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
     </servlet>
+    <servlet>
+        <servlet-name>getAllUsersController</servlet-name>
+        <servlet-class>mate.controller.DisplayDriversController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>createDriverController</servlet-name>
+        <servlet-class>mate.controller.CreateDriverController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>createCarController</servlet-name>
+        <servlet-class>mate.controller.CreateCarController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>createNewManufacturerController</servlet-name>
+        <servlet-class>mate.controller.CreateManufacturerController</servlet-class>
+    </servlet>
+    <servlet>
+        <servlet-name>addDriverToCarController</servlet-name>
+        <servlet-class>mate.controller.AddDriverToCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>getAllUsersController</servlet-name>
+        <url-pattern>/driver/all</url-pattern>
+    </servlet-mapping>
```

Comment:
it's better to place servlet-mapping after servlet(it is more easy to read code)